tests/dispenser/TestLimitCommand.test_limit_command_success.False.False.approved.txt

```diff
@@ -0,0 +1,2 @@
+DEBUG: HTTP Request: GET https://snapshottest.dispenser.com/fund/0/limit "HTTP/1.1 200 OK"
+Remaining daily fund limit: 1000000
```

Comment:
Should we add `μAlgo` like we do now when printing the successfully funded message?